src/Layouts/NO_QWERTY.ts

```diff
@@ -0,0 +1,43 @@
+import {LayoutMap} from './Layout';
+
+export const id = 'NO QWERTY';
+export const map: LayoutMap = {
+    'shift+1' : '!',
+    'shift+2' : '"',
+    'shift+3' : '#',
+    'shift+4' : '¤',
+    'shift+5' : '%',
+    'shift+6' : '&',
+    'shift+7' : '/',
+    'shift+8' : '(',
+    'shift+9' : ')',
+    'shift+0' : '=',
+
+    'vk_oem_1' : 'ø',
+    'vk_oem_2' : '-',
+    'vk_oem_3' : '|',
+    'vk_oem_4' : 'å',
+    'vk_oem_5' : '\'',
+    //PROBLEM: Dead key 'vk_oem_6' : '¨',
+    'vk_oem_7' : 'æ',
+    'vk_oem_102' : '<',
+
+    'shift+vk_oem_1' : 'Ø',
+    'shift+vk_oem_2' : '_',
+    'shift+vk_oem_3' : '§',
+    'shift+vk_oem_4' : 'Å',
+    'shift+vk_oem_5' : '*',
+    //PROBLEM: Dead key 'shift+vk_oem_6' : '^',
+    'shift+vk_oem_7' : 'Æ',
+    'shift+vk_oem_102' : '>',
+
+    'vk_oem_minus'  : '+',
+    'vk_oem_plus'   : '\\',
+    'vk_oem_comma'  : ',',
+    'vk_oem_period' : '.',
+
+    'shift+vk_oem_minus'  : '?',
+    'shift+vk_oem_plus'   : '+',
```

Comment:
Yes, you are correct. But, that one's also a dead key. Have commented it out, and commented that it's a dead key.